docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,208 @@
+
+### Configuration
+
+After the [required services](#requirements) are deployed, you're ready to start the `Airy apps` inside a Kubernetes. Connecting the `Airy apps` to the Kafka cluster, PostgreSQL and Redis can be done by creating a configuration file, prior to deploying the apps. Make sure that the `Airy apps` also have network connectivity to the required services.
+
+The file `infrastructure/airy.conf.all` contains an example of all possible configuration parameters. This file should be copied to `airy.conf` and edited according to the particular environment.
+
+```sh
+cd infrastructure
+cp airy.conf.all airy.conf
+```
+
+Edit the file to configure connections to the base services. Make sure that the following sections are configured correctly, so that the `Airy apps` to start properly:
+
+```
+apps:
+  kafka:
+    ...
+  redis:
+    ...
+  postgresql:
+    ...
+```
+
+### Deployment
+
+We provided a Helm chart to deploy all of the `Airy apps`. Before you can run helm, you must configure the system via the `airy.conf` file. After that run:
+
+```sh
+cp airy.conf ./helm-chart/charts/apps/values.yaml 
+helm install airy-apps ./helm-chart/charts/apps/ --timeout 1000s
+```
+
+By default, the `Airy apps` deployments start with `replicas=0` so to scale them up, run:
+```sh
+kubectl scale deployment -l airy=api --replicas=1
+kubectl scale deployment -l airy=sources --replicas=1
+kubectl scale deployment -l airy=webhook --replicas=1
+kubectl scale deployment -l airy=frontend --replicas=1
+```
+
+At this point you should have a running `Airy Core Platform` in your environment 🎉.
+
+To deploy with a different `image tag` (for example `beta` from the `develop` branch), you can run:
+```sh
+export AIRY_VERSION=beta
+helm install airy-apps ./helm-chart/charts/apps/ --set global.appImageTag=${AIRY_VERSION} --timeout 1000s
+```
+
+If afterwards you need to modify or add other config parameters in the `airy.conf` file, after editing the file run:
+
+```sh
+cp airy.conf ./helm-chart/charts/apps/values.yaml
+helm upgrade airy-apps ./helm-chart/charts/apps/ --timeout 1000s
+```
+
+If you have deployed the Airy Core Platform with a specific version tag, export the `AIRY_VERSION` variable, before running `helm upgrade`:
+
+```sh
+cp airy.conf ./helm-chart/charts/apps/values.yaml
+export AIRY_VERSION=beta
+helm upgrade airy-apps ./helm-chart/charts/apps/ --set global.appImageTag=${AIRY_VERSION} --timeout 1000s
+```
+
+## Network
+
+### Sources
+
+The helm chart creates separate NodePort service for every source with the naming convention  `sources-SOURCE_NAME-webhook`. These services need to be exposed publicly on the Internet, so that the sources can send messages and events to the webhook services, inside the kubernetes cluster.
```

Comment:
```suggestion
The helm chart creates separate NodePort service for every source with the naming convention  `sources-SOURCE_NAME-webhook`. These services must be exposed publicly on the Internet, so that the sources can send messages and events to the webhook services, inside the Kubernetes cluster.
```